src/main/java/org/wise/portal/presentation/web/controllers/student/StudentAPIController.java

```diff
@@ -336,6 +337,10 @@ ResponseEntity<Map<String, Object>> createStudentAccount(
       @RequestBody Map<String, String> studentFields, HttpServletRequest request)
       throws DuplicateUsernameException, InvalidNameException {
     StudentUserDetails sud = new StudentUserDetails();
```

Comment:
You can move `StudentUserDetails sud = new StudentUserDetails();` down to just before the line `sud.setFirstname(firstName);` since we don't use that variable until then.